pkg/visor/config.go

```diff
@@ -5,39 +5,30 @@ import (
 	"errors"
 	"fmt"
 	"io/ioutil"
-	"net"
-	"os"
-	"path/filepath"
 	"sync"
 	"time"
 
-	"github.com/SkycoinProject/dmsg"
+	"github.com/SkycoinProject/skywire-mainnet/pkg/app/launcher"
+	"github.com/SkycoinProject/skywire-mainnet/pkg/restart"
+
 	"github.com/SkycoinProject/dmsg/cipher"
-	"github.com/SkycoinProject/dmsg/dmsgpty"
 	"github.com/SkycoinProject/skycoin/src/util/logging"
 
-	"github.com/SkycoinProject/skywire-mainnet/pkg/app/appcommon"
 	"github.com/SkycoinProject/skywire-mainnet/pkg/routing"
 	"github.com/SkycoinProject/skywire-mainnet/pkg/skyenv"
 	"github.com/SkycoinProject/skywire-mainnet/pkg/snet"
-	"github.com/SkycoinProject/skywire-mainnet/pkg/transport"
-	trClient "github.com/SkycoinProject/skywire-mainnet/pkg/transport-discovery/client"
 )
 
 //go:generate readmegen -n Config -o ./README.md ./config.go
 
 const (
 	// DefaultTimeout is used for default config generation and if it is not set in config.
+	// TODO: Put this in skyenv.
 	DefaultTimeout = Duration(10 * time.Second)
-	// DefaultLocalPath is used for default config generation and if it is not set in config.
-	DefaultLocalPath = "./local"
-	// DefaultAppsPath is used for default config generation and if it is not set in config.
-	DefaultAppsPath = "./apps"
-	// DefaultLogLevel is used for default config generation and if it is not set in config.
-	DefaultLogLevel = "info"
-	// DefaultSTCPPort ???
-	// TODO: Define above or remove below.
-	DefaultSTCPPort = 7777
+
+	// ConfigVersion of the visor config.
+	// TODO: Put this in skyenv?
+	ConfigVersion = "v1.0.0"
```

Comment:
I think we need to decide if we are going to use it. If so, it's worth modifying when the config is changed.

Reply:
@nkryuchkov I'm thinking of moving config into a separate module, maybe `/pkg/visor/visorconfig` and having versioned structs. E.g. `visorconfig.V1` or something.

Reply:
@evanlinjin That sounds like a good idea.